03-lesson-plan.Rmd

```diff
@@ -1,3 +1,11 @@
 # Lesson Plan
 
-Coming soon!
+| Activity | Undergraduate course | Graduate course | |
+| :-- | :-- | :-- | :--|
+| Getting started [_Logging into AnVIL_](https://jhudatascience.org/GDSCN_Book_SARS_Galaxy_on_AnVIL/student-activity-guide.html#set-up), [_Provisioning Galaxy workspace_](https://jhudatascience.org/GDSCN_Book_SARS_Galaxy_on_AnVIL/student-activity-guide.html#starting-galaxy) | 10-15 minutes | 10-15 minutes | _Provisioning will take 8-10 minutes of waiting_ |
```

Comment:
I think some of these links are redirecting.. Might be easiest to just copy the link at the light blue pound sign for the url you are thinking of! 
![image](https://user-images.githubusercontent.com/15618412/159807498-007cefef-d37d-4584-9d98-8adb0b181ceb.png)

Reply:
Yeah, it's not quite right yet. (And sorry for sending the commit notice - I haven't quite figured out how to view the preview using R without submitting it)

Reply:
I am waiting to edit this commit until after the exercises have been renumbered (#39). I am not sure how editing the student guide is going to change the links in the lesson plan. I'd assumed git would automatically make the changes, but I didn't find verification.